requirements/eval-deps.pip

```diff
@@ -7,7 +7,7 @@ fishhook~=0.3; python_version == '3.12'
 forbiddenfruit~=0.1
 fuzzywuzzy~=0.18
 lark~=1.2
-matplotlib~=3.9
+matplotlib~=3.9 ; python_version != '3.13'
```

Comment:
I take it there's (currently) no way to only exclude nogil builds?

Reply:
unfortunatly not right now.
https://discuss.python.org/t/environment-marker-for-free-threading/60007/5
I'll force push the commit to link to this discourse thread